samples/matter/common/src/bridge/bridge_storage_manager.cpp

```diff
@@ -211,11 +233,27 @@ bool BridgeStorageManager::LoadBridgedDevice(BridgedDevice &device, uint8_t inde
 	counter += sizeof(device.mEndpointId);
 	memcpy(&device.mDeviceType, buffer + counter, sizeof(device.mDeviceType));
 	counter += sizeof(device.mDeviceType);
+	memcpy(&device.mUniqueIDLength, buffer + counter, sizeof(device.mUniqueIDLength));
```

Comment:
This is not going to work if someone has older bridge version with some bridged devices stored in the memory and then he does DFU, right? As unique id will not be present for them. I'm starting to wonder, shouldn't we update the data structure version from `1` to `2`. See the `MigrateData` that handles only no version case (initial version), and the version == 1 that was the first time we had to modify the structure. Now we should probably create case for version `2` and provide a migration that will generate unique id and shift all data.

Reply:
I will do this in a follow-up task.